_delphi_utils_python/delphi_utils/validator/validate.py

```diff
@@ -63,5 +63,5 @@ def validate(self):
             all_frames = []
         else:
             all_frames = aggregate_frames(frames_list)
-        self.dynamic_validation.validate(all_frames, report)
+            self.dynamic_validation.validate(all_frames, report)
```

Comment:
suggestion: all_frames isn't needed when frames_list is empty

could we replace the entire if block with something like this instead?

```
if len(frames_list) > 0:
    self.dynamic_validation.validate(aggregate_frames(frames_list), report)

```

Reply:
That makes sense and would look much cleaner. I'll put that in and do some testing.